prime-router/metadata/fhir_mapping/hl7/datatype/CodeableConcept.yml

```diff
@@ -57,14 +73,15 @@ coding_6:
     display: String, CWE.2 | CNE.2 | CE.2
     system: $overrideSystem | CWE.3 | CNE.3 | CE.3
     version: String, CWE.7 | CNE.7
+    codingSystemOID: String, CWE.14
```

Comment:
(optional) Thoughts on adding the `codeIndex: "identifier"` here? Even though it's the default, it lets all the CWE fields here follow the same pattern and makes the `codeIndex` obvious here without looking at the other file.

Reply:
Will add!